statistics/histogram.go

```diff
@@ -738,7 +738,7 @@ func (c *Column) equalRowCount(sc *stmtctx.StatementContext, val types.Datum) (f
 		return 0.0, nil
 	}
 	if c.NDV > 0 && c.outOfRange(val) {
-		return c.totalRowCount() / (float64(c.NDV)), nil
+		return float64(modifyCount) / float64(c.NDV), nil
```

Comment:
how about: 
- `modifyCount* (1/(NDV+1))`, or:
- `modifyCount* (1-80%)`

Reply:
For the first one, what is the benefit to add 1? For the second one, I think it is better to make use of the `NDV` info.

Reply:
never mind